recipe/meta.yaml

```diff
@@ -1,50 +1,102 @@
 {% set name = "polars" %}
-{% set version = "0.18.3" %}
+{% set version = "0.20.18" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polars-{{ version }}.tar.gz
-    sha256: efc3f629fddb060dc90c2dbd575824c1d662f37294c18c769d28e2bc4d5a1413
+    sha256: 8a321cbdbb459e3c0cc1af2ce6ac930d0d3b5ccbeb2dd3e4237ad07d487fd290
 
 build:
   number: 0
-  skip: true  # [py<37]
```

Comment:
I would keep the skip and make it explicit that it only supports 3.8+

Reply:
done